tests/target/issue-5244/wrapped.rs

```diff
@@ -0,0 +1,8 @@
+// rustfmt-wrap_comments: true
+
+/// Here is me writing some documentation that is too long oh me oh my now some
+/// code please! 
```

Comment:
@grant0417 There's trailing whitespace here. That might explains the test failures. rustfmt definitely shouldn't introduce trailing whitespace here.

Reply:
Forgot to run the tests locally, this is fixed now.